sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/FileSASTests.groovy

```diff
@@ -201,6 +204,7 @@ class FileSASTests extends APISpec {
     def "FileSAS network test upload fails"() {
         setup:
         String data = "test"
+        primaryShareClient.create()
```

Comment:
Need to update playback record, this introduces an additional network call,

Reply:
Updated